util/maintain-banners.js

```diff
@@ -3,8 +3,7 @@ const globby = require('globby');
 const path = require('path');
 
 const bannerSourcePath = path.join(__dirname, 'license-template.txt');
-const files = globby.sync(path.join(__dirname, '..','packages/*/{index.js,index.ts,lib.js,src/**/*.{js,ts},test/integration-test/*.{js,ts},test/e2e/**/*.{js,ts}}'));
-// console.log(files);
+const files = globby.sync(path.join(__dirname, '..','{src/**/*.js,test/**/*.js,dist/bundles/*.js}'));
```

Comment:
There's a logic here, perhaps we can explain it in a comment?  All JS files of ours?